spec/pluck_to_tstruct_spec.rb

```diff
@@ -58,8 +58,8 @@ class WizardWithWandT < T::Struct
     include TStructComparable
 
     const :name, String
-    const :house, String
     const :wand_wood_type, String
+    const :house, String
```

Comment:
With this change in the order of the tstruct props, the spec on line 138 fails without the new changes to `pluck_to_tstruct.rb`.

https://github.com/chanzuckerberg/sorbet-rails/blob/447ecbe4a2586ce8aa9055290d571fe34a55e27f/spec/pluck_to_tstruct_spec.rb#L138-L147

Rather than the expected array, the actual array would look like:

```
[
     WizardWithWandT.new(name: "Harry Potter", house: "Holly", wand_wood_type: "Gryffindor"),
     WizardWithWandT.new(name: "Hermione Granger", house: "Vine", wand_wood_type: "Gryffindor"),
]
```